src/database/engine/journalfile.c

```diff
@@ -669,6 +669,7 @@ static void journalfile_restore_extent_metadata(struct rrdengine_instance *ctx,
     }
 
     time_t now_s = max_acceptable_collected_time();
```

Comment:
I am going to put some warnings that I am receiving for this file, but they do not have relationship with commits:

```sh
/home/thiago/Netdata/tests_netdata/src/database/engine/journalfile.c: In function 'journalfile_create':
/home/thiago/Netdata/tests_netdata/src/database/engine/journalfile.c:595:30: warning: 'superblock' may be used uninitialized [-Wmaybe-uninitialized]
  595 |     (void) strncpy(superblock->version, RRDENG_JF_VER, RRDENG_VER_SZ);
      |                    ~~~~~~~~~~^~~~~~~~~
/home/thiago/Netdata/tests_netdata/src/database/engine/journalfile.c:576:26: note: 'superblock' was declared here
  576 |     struct rrdeng_jf_sb *superblock;
      |                          ^~~~~~~~~~
In function 'journalfile_check_superblock',
    inlined from 'journalfile_load' at /home/thiago/Netdata/tests_netdata/src/database/engine/journalfile.c:1552:11:
/home/thiago/Netdata/tests_netdata/src/database/engine/journalfile.c:630:11: warning: 'superblock' may be used uninitialized [-Wmaybe-uninitialized]
  630 |     iov = uv_buf_init((void *)superblock, sizeof(*superblock));
      |           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/thiago/Netdata/tests_netdata/src/database/engine/journalfile.c: In function 'journalfile_load':
/home/thiago/Netdata/tests_netdata/src/database/engine/journalfile.c:622:26: note: 'superblock' was declared here
  622 |     struct rrdeng_jf_sb *superblock;
      |                          ^~~~~~~~~~
In function 'journalfile_iterate_transactions',
    inlined from 'journalfile_load' at /home/thiago/Netdata/tests_netdata/src/database/engine/journalfile.c:1562:14:
/home/thiago/Netdata/tests_netdata/src/database/engine/journalfile.c:828:15: warning: 'buf' may be used uninitialized [-Wmaybe-uninitialized]
  828 |         iov = uv_buf_init(buf, size_bytes);
      |               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
I will look at those in a different PR